gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/web/internal/configureWebApplication.kt

```diff
@@ -23,13 +23,34 @@ internal fun Project.configureWeb(
     composeExt: ComposeExtension,
 ) {
     val webExt = composeExt.extensions.getByType(WebExtension::class.java)
+
+    // here we check all dependencies (including transitive)
+    // If there is compose.ui, then skiko is required!
+    val shouldRunUnpackSkiko = project.provider {
+        var dependsOnComposeUi = false
+        project.configurations.matching { configuration ->
+            val isWasmOrJs = configuration.name.contains("js", true) ||
+                    configuration.name.contains("wasm", true)
+
+            configuration.isCanBeResolved && isWasmOrJs
+        }.all { configuration ->
+            val match = configuration.incoming.artifacts.resolvedArtifacts.get().any { artifact ->
+                artifact.id.componentIdentifier.toString().contains("org.jetbrains.compose.ui")
```

Comment:
it also includes "ui-unit". can we check the exact match with ui module?

Reply:
done